macro-and-me/src/pages/FoodSearch.jsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from 'react';
+
+export default function FoodSearch() {
+  const [query, setQuery] = useState('');
+  const [results, setResults] = useState([]);
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState('');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    if (query === '') return;
+
+    setLoading(true);
+    setError('');
+
+    try {
+      const apiKey = process.env.REACT_APP_USDA_API_KEY; // USDA API Key from env
```

Comment:
How the env variables will be passed to React app? If through .env file, please install dotenv module and provide ".env_example" file.